src/main/webapp/app/overview/course-conversations/layout/conversation-thread-sidebar/conversation-thread-sidebar.component.scss

```diff
@@ -62,6 +62,15 @@
     }
 }
 
+.expanded-thread.expanded {
+    width: 100%;
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    z-index: 1050;
```

Comment:
What's the rationale for this z-index value here?

Reply:
The z-index prevents overlap with the search bar, ensuring that the search bar remains visible on top. However, a lower value could also achieve this. I can adjust it in a follow-up PR.

Reply:
The z-index prevents the search bar from covering the thread view, as it would otherwise take its place. However, a lower value could also achieve this. I can adjust it in a follow-up PR.